securedrop/source_templates/generate.html

```diff
@@ -21,7 +21,7 @@ <h2>{{ gettext('Get Your Codename') }}</h2>
 <form id="create-form" method="post" action="/create" autocomplete="off">
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
   <input name="tab_id" type="hidden" value="{{ tab_id }}">
-  <button type="submit" aria-label="{{ gettext('Continue') }}">
+  <button type="submit" aria-label="{{ gettext('Continue to Submissions Page') }}">
```

Comment:
Nit:  It looks like the most-canonical term we have for this page is "submission page", singular, per <https://docs.securedrop.org/en/stable/source.html#making-your-first-submission>.  (Though I would bikeshed in favor of "submission form"!)

```suggestion
  <button type="submit" aria-label="{{ gettext('Continue to Submission Page') }}">
```